docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -201,13 +235,36 @@ The following configuration properties are available:
      - Base64-encoded secret key used to encrypt the generated token.
        By default it's generated during startup.
 
+.. _trino-oauth2-troubleshooting:
 
 Troubleshooting
 ---------------
 
-If you need to debug issues with Trino OAuth 2.0 configuration you can change
-the :ref:`log level <log-levels>` for the OAuth 2.0 authenticator:
+To debug issues, change the :ref:`log level <log-levels>` for the OAuth 2.0
+authenticator:
 
 .. code-block:: none
 
     io.trino.server.security.oauth2=DEBUG
+
+To debug issues with OAuth 2.0 authentication use with the web UI, set the
+following configuration property:
+
+.. code-block:: none
+
+    io.trino.server.ui.OAuth2WebUiAuthenticationFilter=DEBUG
+
+This assumes the OAuth 2.0 authentication for the Web UI is enabled as described
+in :ref:`trino-server-configuration-oauth2`.
+
+The logged debug error for a lapsed refresh token is ``Tokens refresh challenge
+has failed``.
+
+.. warning::
+
+    If a refresh token lapses, the user session is interrupted and the user must
+    reauthenticate by logging in again. Be sure to set the
```

Comment:
should we say "authenticate again" only or "reauthenticate by logging in" .. seems like we duplicating now

Reply:
Also .. "Be sure" ... can we find something better

Reply:
Ensure?